src/games/strategy/triplea/ai/proAI/data/ProOtherMoveOptions.java

```diff
@@ -52,6 +54,7 @@ public String toString() {
 
   private static Map<Territory, ProTerritory> createMaxMoveMap(final List<Map<Territory, ProTerritory>> moveMaps,
       final PlayerID player, final boolean isAttacker) {
+    final GameData data = ProData.getData();
```

Comment:
Please minimize vertical distance between usage and declaration. The repeated method call is not a performance concern. Reducing the vertical distance reduces mental burden to understand this method, it's just one more thing to remember when looking at hte for loop.